### ISSUE TITLE
I haven't received a CodeTriage email in a few days :disappointed: 

### ISSUE THREAD

**Issue by flavorjones** [codetriage/CodeTriage#1619]
Hello! And thank you for CodeTriage. I've halved the number of open issues on Nokogiri over the past two months with your help!

I realized this morning that the last CodeTriage email I received was on Jan 24th. Previously I had been receiving one a day, regularly. There doesn't appear to be any filtering going on in my email client, though I suppose and fear that something like inadvertent blocklisting codetriage.com emails may have happened (I use Gmail).

## Expected Behavior

I would love to continue to receive a daily email.

## Current Behavior

No email. :cry: 

## Possible Solution

:shrug:

**Comment by flavorjones** [comment]
Today I received an email. It's February 1st, maybe this has to do with monthly limits from the email delivery service?

**Comment by schneems** [comment]
Strange. Did you go awhile without interacting with an email? We have some auto back-off logic that kicks in if we see that emails aren’t being clicked. It might be that back off logic.

**Comment by schneems** [comment]
Looking at your account in the DB it looks like code triage thinks it's been 5 days since you've clicked on a link in one of the emails:

```
irb(main):007:0> job.send(:skip?, user)
=> "Email backoff"
irb(main):010:0> user.days_since_last_clicked
=> 5
irb(main):011:0> user.email_frequency
=> "daily"
```

Here's where the backoff logic is called https://github.com/codetriage/CodeTriage/blob/51d496085bc076b43302e1c29922271688dacfe0/app/jobs/send_daily_triage_email_job.rb#L61-L63 and here's where that class lives https://github.com/codetriage/CodeTriage/blob/51d496085bc076b43302e1c29922271688dacfe0/app/models/email_rate_limit.rb.

I went ahead and forced it to deliver an email right now, let me know if you didn't get it.

**Comment by flavorjones** [comment]
I got the email! Here's the latest view into my inbox:

[image: image.png]

I believe I didn't click the email on the 15th. Possibly also on the 11th,
which would match the "grow your streak" number.

However, I just want to point out that the streak continued to increase
between the Jan 24 and Feb 1 email ... that's the period when I didn't
receive any emails that prompted me to open this issue. I was clicking on
those email links, but there was a break in delivery for some reason.

**Comment by flavorjones** [comment]
I got the email! Here's the latest view into my inbox:

![image (2)](https://user-images.githubusercontent.com/8207/154810968-8b11dba5-528f-42e5-a3ee-c445ac3d7374.png)

I believe I didn't click the email on the 15th. Possibly also on the 11th, which would match the "grow your streak" number.

However, I just want to point out that the streak continued to increase between the Jan 24 and Feb 1 email ... that's the period when I didn't receive any emails that prompted me to open this issue. I was clicking on those email links, but there was a break in delivery for some reason.

**Comment by tunisiano187** [comment]
Same problem for me, but i see on the codetriage page that the repos have 0 issues, but on the github page, they have 22issues...
Here is an example : https://github.com/chocolatey-community/chocolatey-packages/issues

**Comment by flavorjones** [comment]
FWIW, I believe the situation that caused me to file this original ticket was: there were no more issues to email me about! I had received emails for all the open issues on all the repositories I was subscribed to.

Since then, when a new issue is created, Code Triage has reliably emailed me about it. But if there are no new issues, I won't get any emails.

If my diagnosis is correct, I think this is operating as designed? Might that be what's going on for you, too @tunisiano187 and @ananthvyas ?

**Comment by tunisiano187** [comment]
@flavorjones, i don't think so, the repository has new tickets nearly everyday, and i just ask for 1 per day.
I've tryied to "untriage" then recreate it, and the repositories i've made that are now OK, maybe it's linked to the github token? or a change that needs to be updated?
today i receive 2 triage for issues that are 2 days old
